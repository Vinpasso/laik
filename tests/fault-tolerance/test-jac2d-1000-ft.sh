#!/bin/sh

# $1 should contain the source, $2 the binary directory
echo "$1/checkpoint-jac2d-recovery, $2"
mpirun --oversubscribe -n 8 "$2/checkpoint-jac2d-recovery" --failureCheckFrequency 10 --checkpointFrequency 10 --redundancyCount 1 --rotationDistance 1 --plannedFailure 1 15 --plannedFailure 2 21  --plannedFailure 3 32 --plannedFailure 5 33 -s 1000 > test-jac2d-1000-ft.out
diff -y test-jac2d-1000-ft.out "$1/test-jac2d-1000-ft.expected"
RESULT=$?
echo "Result: $RESULT"
exit $RESULT