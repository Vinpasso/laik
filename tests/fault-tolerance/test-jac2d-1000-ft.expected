1000 x 1000 cells (mem 16.0 MB), running 50 iterations with 8 tasks
Residuum after  1 iters: 3007147.625000
Residuum after 11 iters: 2377.016846
Oops. Process with rank 1 did something silly on iteration 15. Aborting!
Residuum after 11 iters: 2377.016846
Oops. Process with rank 2 did something silly on iteration 21. Aborting!
Residuum after 21 iters: 150.808462
Residuum after 21 iters: 150.808462
Residuum after 31 iters: 82.356528
Oops. Process with rank 3 did something silly on iteration 32. Aborting!
Oops. Process with rank 5 did something silly on iteration 33. Aborting!
Residuum after 31 iters: 82.356528
Residuum after 41 iters: 53.986431
Global value sum after 50 iterations: 2946003.362703
